wd_helper.au3

```diff
@@ -2469,6 +2471,40 @@ Func _WD_DownloadFile($sURL, $sDest, $iOptions = Default)
 	Return SetError(__WD_Error($sFuncName, $iErr, $sParameters, $iExt), $iExt, $bResult)
 EndFunc   ;==>_WD_DownloadFile
 
+; #FUNCTION# ====================================================================================================================
+; Name ..........: _WD_DownloadAsBinary
```

Comment:
I think the function header should be `; #INTERNAL_USE_ONLY# ======` and the function should be `__WD_DownloadAsBinary` instead of `_WD_DownloadAsBinary`. Actually it does not matter, but it would fulfill the consistency 😇 .

Reply:
If you have any reason to change `_WD_DownloadAsBinary` into internal usage only then you should try to apply the same reason to `_WD_DownloadFile` and see if this reason also makes sense here.

If you say that `_WD_DownloadFile`  save data to file I would say the destination does'nt matter here as I always try to avoid writing data to files on disk. Instead, I process the data in memory and write to the database.

This is because of security reason (i.e. GDPR law) - I simply try to minimize the number of places where data leaves a trace.

Summary: if any user of this UDF needs to use the function of writing data to a file, then the function of writing data to memory without writing it to disk is equally necessary.

Reply:
Understood and fair enough @mlipok 👍 .

Then at least the README.md file also has to be adjusted, right?
I mean the additional function should be described in the README like the other functions too.
Is this usually made by @Danp2 ? I believe he do a "Prep release" commit which will contain such things, right? Or do we as PR authors also adjust these files 🤔 ?

Best regards
Sven

Reply:
> Is this usually made by @Danp2 ? I believe he do a "Prep release" commit which will contain such things, right? Or do we as PR authors also adjust these files 🤔 ?

@Danp2 
What you think about ?

Reply:
@Danp2 can you look here ?